documentation/sections/breaking_changes_argument_parsing_and_shebang_lines.md

```diff
@@ -43,37 +31,44 @@ additional '--' before the path to your script.
 
 '--' will be removed from the argument list.
 The positional arguments were: ["path/to/script.coffee","--"]
-...
+hello world!
 ```
 
+Note that the script is still run as usual, printing `hello world!` after the warning text.
+
 #### Shebang scripts
 
+##### Correct Example
+
 On non-Windows platforms, a `.coffee` file can be made executable by adding a shebang (`#!`) line at the top of the file and marking the file as executable. For example:
 
-`executable-script.coffee`:
+*executable-script.coffee:*
 ``` coffeescript
 #!/usr/bin/env coffee
 
 x = 2 + 2
 console.log x
 ```
-
+*output:*
 ``` bash
 > chmod +x ./executable-script.coffee
 > ./executable-script.coffee
 4
 ```
 
-Due to a bug in the argument parsing of previous CoffeeScript versions, this used to fail when trying to pass arguments to the script. Some users on OS X worked around the problem by using `#!/usr/bin/env coffee --` at the top of the file instead. However, that won’t work on Linux, which cannot parse shebang lines with more than a single argument. While these scripts will still run on OSX, CoffeeScript will now display a warning before compiling or evaluating files that begin with a too-long shebang line:
+##### Incorrect Example
+
+Due to a bug in the argument parsing of previous CoffeeScript versions, this used to fail when trying to pass arguments to the script. Some users on OSX worked around the problem by using `#!/usr/bin/env coffee --` at the top of the file instead. However, that won’t work on Linux, which cannot parse shebang lines with more than a single argument. While these scripts will still run on OSX, CoffeeScript will now display a warning before compiling or evaluating files that begin with a too-long shebang line:
```

Comment:
I don't understand what the issue is -- should I add quotes around `OSX`? The space between `OS` and `X` was removed everywhere.